tile/TileRTL.py

```diff
@@ -23,7 +24,7 @@
 from ..noc.ChannelRTL import ChannelRTL
 from ..rf.RegisterRTL import RegisterRTL
 # from ..noc.BypassChannelRTL      import BypassChannelRTL
-
+from py_markdown_table.markdown_table import markdown_table

```

Comment:
Move this after `from pymtl3 import *`, we are sorting these in alphabet ordering.

Reply:
Done, py_ should be before pymtl3 when optimize.